pkg/report/writer.go

```diff
@@ -37,7 +37,8 @@ var (
 )
 
 type Option struct {
-	AppVersion string
+	AppVersion   string
+	DfctlVersion string
```

Comment:
dfctl versions change frequently. should this instead be dfscan?
In case of webhook, we're not using dfctl, dfscan can be an independent entity

Reply:
In case of webhook, we will use dfctl as we discussed this morning. Akash is going to fix it.